src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,104 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import calculator.StringSplitParser;
+
+public class Lotto {
+	public static final String MESSAGE_NOT_ALLOW_LENGTH = "LOTTO_NUMBERS_LENGTH_MUST_BE_6";
+	public static final String MESSAGE_NOT_ALLOW_DUPLICATION = "LOTTO_NUMBERS_ELEMENT_MUST_BE_INDEPENDENT";
+	public static final int LOTTO_NUMBERS_LENGTH = 6;
+
+	private final List<LottoNumber> lottoNumbers;
+
+	public Lotto(List<LottoNumber> lottoNumbers) {
+		if (lottoNumbers.size() != LOTTO_NUMBERS_LENGTH) {
+			throw new IllegalArgumentException(MESSAGE_NOT_ALLOW_LENGTH);
+		}
+		validateDuplication(lottoNumbers);
+
+		this.lottoNumbers = new ArrayList<>();
+		sortLottoNumbers(lottoNumbers);
+		this.lottoNumbers.addAll(lottoNumbers);
```

Comment:
```suggestion
		this.lottoNumbers = new ArrayList<>(lottoNumbers);
		sortLottoNumbers(this.lottoNumbers);
```

소소하지만 생성자를 통해 초기화해볼 수 있을 것 같네요~